### ISSUE TITLE
URL requesting your username and password.

### ISSUE THREAD

**Issue by umeshPathak** [sitespeedio/browsertime#1353]
I tried supplying username and password using below command **browsertime <url>  --basicAuth username@password --firefox.includeResponseBodies all --iterations 1 --headless --browser firefox** , but still it gives error **Dismissed user prompt dialog: <url> is requesting your username and password**. When I dont use --headless , it asks username and password multiple times on the basic auth dialog box.

**Comment by soulgalore** [comment]
Hi @umeshPathak I tested with  https://httpbin.org/basic-auth and that seems to work for me, can you test that to verify that basic auth works?

**Comment by umeshPathak** [comment]
Yes. it worked for other application's URL. it worked, Might be issue with that url.
